Makefile

```diff
@@ -36,8 +37,9 @@ deb_%: clean docker_build_%
 			--deb-dist $* --deb-priority optional \
 			--name paasta-tools-go --package dist \
 			--description "CLI tools for PaaSTA in Go" \
+			--package dist/$* \
 			bin=/usr/ && \
-		chown -R $(UID):$(GID) bin dist \
+		sudo chown -R $(UID):$(GID) bin dist \
```

Comment:
we should not need this.

We should not need to chown nor should we need sudo

Reply:
Hmm.....it's owned by root so far. I need to figure out why.